README.md

```diff
@@ -1,75 +1,100 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# COVID-19 STATS
 
+> This web application provides statistical insights on COVID-19 cases around the world. The application allows users to search for different countries and clicking a country would display all of the covid statistics for that country including its states, provinces and sub regions if available.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
+- React JS & Redux
+- CSS
+- JavaScript
+- Tailwindcss
 
-[Live Demo Link](https://livedemo.com)
+## Live Demo
+|Netlify cloud hosting|
+|----------------------|
+|[View live demo](https://covid19stattistics.netlify.app/)|
 
+## Demo Video link
+|Link to demo video|
+|----------------------|
+|[Demo video link](https://www.loom.com/share/77314569f7184582ac586dd270db9491)|
```

Comment:
- [ ] Good job making all the changes, your app looks great. But for the great work you did so far, it would be nice if you could redo the video and please take note of all the points listed in the project requirement. The previous reviewer did not highlight the points you were missing out in the video but I think it would be good if you could make a better video presentation of this nice project. 👍  

- Recorded a video presenting the features of the project. (You can introduce yourself here, and talk about the main requirements of the project - Kind of like, my name is ...., and this is my capstone project on React in which I am required to build an app that fetches metrics data from an API, render them to the screen and show details for each. That's a good intro to the project and its features.) 👍 
- The maximum length of the video is 5 minutes. 
- The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. (With that intro I made, you've done all this part.) 👍 
- The video contains a demo of the project, highlighting all the project features. (Now demo the features you talked about, click on a country, search and show details of a country in a brief) 👍 
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making. ( You did great work and it would be nice if you could point to a code section that you really liked or are proud of implementing in the project, as this is your work and you are proud of it) 👍 

- [ ] Please consider all these sections above to have a good presentation of your project before you make the video and then you'll be on your way to approval 👍